GitDiffMargin.Commands/ShimCommandHandler.cs

```diff
@@ -1,52 +1,47 @@
-﻿namespace GitDiffMargin.Commands
-{
-    using System;
-    using Microsoft.VisualStudio;
-    using Microsoft.VisualStudio.Commanding;
-    using Microsoft.VisualStudio.Shell;
-    using IOleCommandTarget = Microsoft.VisualStudio.OLE.Interop.IOleCommandTarget;
-    using OLECMD = Microsoft.VisualStudio.OLE.Interop.OLECMD;
-    using OLECMDEXECOPT = Microsoft.VisualStudio.OLE.Interop.OLECMDEXECOPT;
-    using OLECMDF = Microsoft.VisualStudio.OLE.Interop.OLECMDF;
+using System;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Commanding;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.Shell;
 
+namespace GitDiffMargin.Commands
+{
     internal abstract class ShimCommandHandler<T> : ICommandHandler<T>
         where T : CommandArgs
     {
-        private readonly Guid _commandSet;
         private readonly uint _commandId;
+        private readonly Guid _commandSet;
 
         protected ShimCommandHandler(Guid commandSet, uint commandId)
         {
             _commandSet = commandSet;
             _commandId = commandId;
         }
 
-        public abstract string DisplayName
-        {
-            get;
-        }
+        public abstract string DisplayName { get; }
 
         public virtual CommandState GetCommandState(T args)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            OLECMD[] command = { new OLECMD { cmdID = _commandId } };
+            OLECMD[] command = {new OLECMD {cmdID = _commandId}};
             ErrorHandler.ThrowOnFailure(GetCommandTarget(args).QueryStatus(_commandSet, 1, command, IntPtr.Zero));
-            if ((command[0].cmdf & (uint)OLECMDF.OLECMDF_SUPPORTED) == 0)
+            if ((command[0].cmdf & (uint) OLECMDF.OLECMDF_SUPPORTED) == 0)
                 return CommandState.Unspecified;
-            else if ((command[0].cmdf & (uint)OLECMDF.OLECMDF_ENABLED) == 0)
-                return CommandState.Unavailable;
-            else
-                return CommandState.Available;
+
+            return (command[0].cmdf & (uint) OLECMDF.OLECMDF_ENABLED) == 0
+                ? CommandState.Unavailable
+                : CommandState.Available;
         }
 
         public virtual bool ExecuteCommand(T args, CommandExecutionContext executionContext)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
 
-            return ErrorHandler.Succeeded(GetCommandTarget(args).Exec(_commandSet, _commandId, (uint)OLECMDEXECOPT.OLECMDEXECOPT_DODEFAULT, IntPtr.Zero, IntPtr.Zero));
+            return ErrorHandler.Succeeded(GetCommandTarget(args).Exec(_commandSet, _commandId,
+                (uint) OLECMDEXECOPT.OLECMDEXECOPT_DODEFAULT, IntPtr.Zero, IntPtr.Zero));
```

Comment:
💭 I dislike how some tools seem to think code should read like a book. One of the smartest rules I've observed in StyleCop is the part of rules that says all arguments should be on one line, or each argument should be on a line of its own.